_topics/en/dual-funding.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Dual funding
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Dual funding** is creating a payment channel for LN where both
+  parties can contribute funds.  The underlying protocol, called
+  the *version 2 channel establishment protocol*, may also be used for
+  negotiated opening of single-funded channels, but its motivating
+  purpose is providing support for dual funding.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  [Early analysis][dryja single-funded] of LN determined that it would be
+  significantly easier to build software where the user requesting to
+  open the payment channel contributed all funds to that channel and
+  paid all of its onchain fees, called *single funded channels*.  This
+  prevented attackers from freely or cheaply opening new channels,
+  locking up their counterparty's funds, and then making those victims
+  pay onchain fees to get their money back.
+
+  For spenders, single funded channels work great.  As soon as a channel
+  finishes opening, the user can start spending their funds with all of
+  the speed, efficiency, and privacy benefits of LN.  But receivers who
+  open a new single funded channel can't use it to receive funds until
+  they've spent funds.  This creates problems for merchants who want to
+  accept payments over LN but who aren't yet in a position to pay an
+  equal amount of their costs over LN.
+
+  One solution to this problem is to allow channels to be dual funded,
+  immediately allowing spending in either direction once the channel
+  opens.  Dual funded channels don't need to start with the same amount
+  of funding on both sides, so a merchant who wants to be able to
+  receive a significant amount of bitcoins may only need to contribute a
+  small part of the total channel capacity.
+
+  The dual funded protocol may also be used to open new single-funded
+  channels.  This may have advantages when the participating parties
+  want to use the protocol's ability to communicate node preferences and
+  find mutually acceptable values for various channel parameters.
+
+  After dual funding is available, it may be used in combination with
+  new [proposed node announcements][advertising channel liquidity] that
+  could help buyers and sellers of inbound capacity find each other in a
+  decentralized fashion.
+
+  Dual funding does require each party reveal ownership of one of their
+  UTXOs to the other parties.  Like other protocols where this is
```

Comment:
perhaps s/the other parties/the other party/ . Or maybe "to the proposed counterparty" or similar.